README.md

```diff
@@ -9,6 +9,7 @@ The following steps will help you to setup Sunbird RC and Esignet services using
 
 ## Requirements
 
+* Java 21
```

Comment:
Is this mandatory? docker image will take care of the java run time

Reply:
java 21 is required when jar is build locally, updated in point 9 under custom plugin and removed same from requirements